app.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+runtime: python37
```

Comment:
is app.yaml now at the root of the repo? If so, it needs to be moved to /website

Reply:
Also, you should update the release script at [root]/make_release.sh to deploy the website changes to AppEngine

Reply:
I commented this elsewhere, but I'd prefer if we could trigger this through github actions instead of the release script, to be more transparent. 

Currently the release script just edits some version numbers and pushes a new branch to git. I'd prefer to capture the important automated tasks as part of the CI pipeline so we can watch for errors

Reply:
This sounds good to me - let's do that.